tests/test_daemon.cpp

```diff
@@ -876,8 +921,12 @@ INSTANTIATE_TEST_SUITE_P(Daemon, MinSpaceViolatedSuite,
                                  Values("0", "0B", "0GB", "123B", "42kb", "100")));
 INSTANTIATE_TEST_SUITE_P(Daemon, LaunchImgSizeSuite,
                          Combine(Values("test_create", "launch"),
-                                 Values(std::vector<std::string>{}, std::vector<std::string>{"--disk", "4G"}),
+                                 Values(std::vector<std::string>{}, std::vector<std::string>{"--disk", "4G"},
+                                        std::vector<std::string>{"--disk", "9999G"}),
```

Comment:
Good point, you can request a very large disk size and assume it's not acceptable. That will allow you to drop the QStorageInfo stuff. 

However, 10 terabytes may not be so uncommon in a few years. MemorySize currently uses a `long long`, which is guaranteed to be at least 64 bits. It's currently signed (something we could change), so we need something very large but below `2^63`. Perhaps `"999999999G"`. When 1 exabyte is not enough we need to change the underlying representation anyway :wink:

Reply:
Oh, interesting! Thank you, I've made this change.